shell.nix

```diff
@@ -0,0 +1,27 @@
+let
```

Comment:
If we are gonna add this, I'd add a comment mentioning that the dev team doesn't provide any guarantees around the validity of this config